.vscode/settings.json

```diff
@@ -1,4 +1,5 @@
 // Place your settings in this file to overwrite default and user settings.
 {
-  "stylelint.config": { "extends": "./client/.stylelintrc" }
+  "stylelint.config": { "extends": "./client/.stylelintrc" },
+  "eslint.nodePath": "./client/node_modules"
```

Comment:
I'm not sure I like this in here as not everyone is using vscode, so isn't very project specific. It could live in another repo if you'd like? I don't check in any of my IntelliJ stuff for that reason.

Could it read from an editorconfig file instead, which is generic and can then validate that contributors follow our style?